.github/workflows/run-pre-commit.yaml

```diff
@@ -2,7 +2,7 @@ name: Run pre-commit
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, develop ]
```

Comment:
we no longer have a develop branch - there is only a release (with the corresponding CalVer patter) and a main branch